ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3213,5 +3214,57 @@ def test093_select_large_number_of_test_units_in_one_testplan(self):
             if result['test_plan'] == testPlan['testPlan']['text']:
                 for testunit in testunit_names:
                     self.assertIn(testunit, result['test_units'])
+    
+    @parameterized.expand(['before', 'after'])
```

Comment:
no need to check before/after the saving

Reply:
done